### PR TITLE
Multiple authors for blog posts

### LINKED ISSUE #259: Allow multiple authors for blog posts
A blog post can have multiple authors, in that case all the authors are listed.

### PR DESCRIPTION
**Pull request by fmubaidien**
Resolves #259 
- Added an iterator to go through the authors array instead of just one author
- Added a tilte before listing authors at the bottom of the article
- Changed CSS and positioning to accommodate the changes that come with multiple authors

### CODE REVIEW

**Comment by geokhoury** [comment]
#### 📝Changed routes:


| [**/**](https://portal-web-ben261quf-josa.vercel.app/) |
|:-:|
|<a href="https://portal-web-ben261quf-josa.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/6aIqMppI7M520hdZo9mZXbQPFEVkymnoWxxGILsKPTtCGZZVfTjQGUIr3bk9faIYkBalNFvfj8d4Neh-9KCs1w.png">(view full size)</a> |







Commit 50ea1033866f84cd3ea33a72b5ef0d29bd94a933 (https://portal-web-ben261quf-josa.vercel.app).

File: components/Blog/BlogSingle.vue [hunk 1]
```diff
@@ -8,10 +9,9 @@
         {{ article.publishDate ? article.publishDate: article.created_at | fullDate($i18n.locale) }}</div>
       <p v-if="article.excerpt" class="excerpt" :class="article.language">{{ article.excerpt }}</p>
       <!-- author and Share -->
-      <div class="flex items-center flex-wrap justify-between">
-        <author v-if="article.author" class="mb-4 md:mb-12" :name="article.author['name_' + $i18n.locale]"
-          :picture="article.author.picture" />
-        <shareButtons v-if="url" class="mb-4 md:mb-12" :url="url" />
+        <div v-if="article.authors.length" class="flex flex-wrap flex-row my-4" :dir="$dir()">
```
Comment (mahasneh) [inline components/Blog/BlogSingle.vue:16]: This should have a margin-right (if LTR) or margin-left (if RTL) so there's spacing between authors, add classes like `rtl:ml-6 ltr:mr-6`

File: components/Blog/BlogSingle.vue [hunk 2]
```diff
@@ -8,10 +9,9 @@
...
+          <author v-for="author in article.authors" v-bind:key="author.id" class="mb-4 flex-shrink-0"
```
Comment (mahasneh) [inline components/Blog/BlogSingle.vue:17]: This should have a margin-right (if LTR) or margin-left (if RTL) so there's spacing between authors, add classes like `rtl:ml-6 ltr:mr-6`

File: components/Blog/BlogSingle.vue [hunk 3]
```diff
@@ -5,8 +5,12 @@
     <img v-else class="thumbnail md:ltr:mr-6 md:rtl:ml-6 w-full" :src="placeholderImage" />
     <div ref="content" class="container content">
       <h2 ref="title">{{ article.title }}</h2>
-      <div class="text-josa-warm-grey-dark font-bold my-4">
-        {{ article.publishDate ? article.publishDate: article.created_at | fullDate($i18n.locale) }}</div>
+        <div class="flex flex-row flex-wrap justify-between items-center my-2">
```
Comment (mahasneh) [inline components/Blog/BlogSingle.vue:8]: This used to have a `my-4`, actually more spacing is needed here as the share buttons are quite bulky, make them `my-6`

File: components/Blog/BlogSingle.vue (hunk 2 above)
Comment (wissamataleh) [reply components/Blog/BlogSingle.vue:17]: We have a problem with the directions, that's why margins aren't added yet, we're working on it.

File: components/Blog/BlogSingle.vue [hunk 4]
```diff
@@ -135,7 +135,7 @@
   }
 
   hr {
-    @apply my-12 border-solid border-josa-warm-grey-dark;
+    @apply mt-12 mb-4 border-solid border-josa-warm-grey-dark;
```
Comment (mahasneh) [inline components/Blog/BlogSingle.vue:153]: This changes the style of all horizontal rules? Sure we need this?

**Review by wissamataleh — APPROVED** [review]
Layout, margins, and spacing will be fixed in following PR

**Pull request merged**